front/lib/swr.ts

```diff
@@ -823,6 +827,45 @@ export function useAgentConfigurations({
   };
 }
 
+export function useProgressiveAgentConfigurations({
+  workspaceId,
+}: {
+  workspaceId: string;
+}) {
+  const {
```

Comment:
suprising to me because this doesn't guarantee that we get the most popular here? cc @JulesBelveze I thought we had moved to getting the 24 most popular sIds from redis, then fetching the 24 light agents sIds?

Reply:
@flvndvd did you see this com?

Reply:
As per IRL, it was like this in main already so we'll discuss for another PR